web-nfc/nfc_recordType_empty-manual.https.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset=utf-8>
-<title>Web NFC Test: push and watch empty records</title>
+<title>Web NFC Test: push and read empty records</title>
```

Comment:
Nit: s/push/write

Reply:
@Honry, we need replace `push` with `write` in all descriptions, right?

Reply:
Right.